main.cpp

```diff
@@ -84,42 +104,92 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+float throwHammer(double hammerWeight, double power)
+{
+    ignoreUnused(hammerWeight, power);
+    return {};
+}
 
 /*
  2)
  */
+bool isStuck(bool canMove = false, bool canSee = false, bool canHear = false)
+{
+    ignoreUnused(canMove, canSee, canHear);
+    return {};
+}
 
 /*
  3)
  */
+int yourChange(double cost, double paid, int dollars = 0, int quarters = 0, int dimes = 0)
+{
+    ignoreUnused(cost, paid, dollars, quarters, dimes);
+    return {};
+}
 
 /*
  4)
  */
+float explode(double temp, double pressure, bool fracture = false)
+{
+    ignoreUnused(temp, pressure, fracture);
+    return {};
+}
 
 /*
  5)
  */
+char letterGrade(int totalPoints, int possiblePoints, bool wasLate = false)
```

Comment:
This function name does not contain a verb.
Function names need to describe actions
- [ ] revise